### DISCUSSION TITLE
How to keep IntelliJ in-sync with sdkman installations?

### DISCUSSION THREAD

**Discussion by travisfw** [sdkman/sdkman-cli#1244] [Ideas]
Every time I choose a JDK for a project it has a bunch of out of date info. I wonder if I should write a little script to update an XML file somewhere, to keep the version directory managed by sdkman in-sync with the options available in IntelliJ. Where might that XML file be? Or, I suppose the heavyweight alternative would be to write an IDE plugin. How hard could it be.

**Comment by helpermethod** [comment]
Hi @travisfw!

What do you mean by out-of-date info? IntelliJ should let you choose all Java candidates which have been installed by SDKMAN! (except when you use SDKMAN!s Homebrew version).

**Comment by sombriks** [comment]
not sure if it's a sync issue, but intellij says that it can do that: https://www.jetbrains.com/help/idea/sdk.html#configure-jdk-with-version-manager
